pulp_rpm/tests/functional/constants.py

```diff
@@ -1542,9 +1543,347 @@
     "name",
     "packages",
     "profiles",
-    "pulp_created",
-    "pulp_href",
-    "static_context",
-    "stream",
+    "pulp_created" "stream",
     "version",
 ]
+
```

Comment:
It would be great if we could have things like the following modulemds be static asset-files that the tests load, rather than having them embedded in the test-code. We have (iirc) similar embedding for advisory-tests. 

That's not something to do under this PR, just seeing them here brought to mind.

Reply:
Well for the publish test we do compare the fixture file directly against the Pulp file, but in this case we're looking at how the API output is structured, and the file can't help us with that.  Or do you mean we have JSON for the advisory tests laying around somewhere?

Reply:
> Or do you mean we have JSON for the advisory tests laying around somewhere?

This - get the data out of the code. Among other things, makes it easier to produce/manipulate said data using JSON tools, than when it's embedded inside python.

Anyway - not a thing for this PR, just something that's been itching at me that these changes brought back to mind.